src/robot/model/testsuite.py

```diff
@@ -155,12 +155,12 @@ def metadata(self, metadata: 'Mapping[str, str]|None') -> Metadata:
         return Metadata(metadata)
 
     @setter
-    def suites(self, suites: 'Sequence[TestSuite|DataDict]') -> 'TestSuites':
-        return TestSuites(self.__class__, self, suites)
+    def suites(self, suites: 'Sequence[TestSuite|DataDict]') -> 'TestSuites[TestSuite]':
+        return TestSuites["TestSuite"](self.__class__, self, suites)
```

Comment:
We've generally used single quotes with strings (unless the string itself contains them). For consistency it would be good to use it here as well.